draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -369,8 +364,8 @@ steps, taking the `stream` as input.
 
 1. Assert: `metadata` has an entry with the key "primaryUrl".
 
-1. If `metadata` doesn't have entries with keys "requests" and "manifest",
-   return a "format error" with `fallbackUrl`.
+1. If `metadata` doesn't have entries with key "requests", return a
```

Comment:
s/entries/an entry/
(sorry for not noticing in the previous round)

Reply:
done